src/FileHelper.php

```diff
@@ -180,26 +180,27 @@ public static function clearDirectory(string $directory, array $options = []): v
      * Removes a file or symlink in a cross-platform way.
      *
      * @param string $path
-     *
-     * @return bool
      */
-    public static function unlink(string $path): bool
+    public static function unlink(string $path): void
     {
         $isWindows = DIRECTORY_SEPARATOR === '\\';
 
         if (!$isWindows) {
-            return unlink($path);
+            unlink($path);
+            return;
         }
 
-        if (is_link($path) && is_dir($path)) {
-            return rmdir($path);
+        if (is_link($path)) {
+            if (false === @unlink($path)) {
+                @rmdir($path);
```

Comment:
```suggestion
                rmdir($path);
```

Because we don't want to stay silent in case it's windows and we fail to unlink.